assignment/week02/assign2/index.js

```diff
@@ -0,0 +1,207 @@
+let history_list=[
+    {
+        category:"과외비",
+        detail:"10월 월급",
+        amount: 500000
+    },{
+        category:"식비",
+        detail:"서브웨이",
+        amount:-30000
+    },{
+        category:"용돈",
+        detail:"10월 용돈",
+        amount:100000
+    },{
+        category:"식비",
+        detail:"수육국밥",
+        amount:-10000
+    }];
```

Comment:
컨벤션을 어떻게 정하냐에 따라 달라지겠지만,
보통 상수값은 `대문자`로 표시해줘서 이 값은 상수값이다!라는 점을 한 눈에 알아볼수 있게 해주는 것이 좋아요!
```suggestion
let HISTORY_LIST=[
    {
        category:"과외비",
        detail:"10월 월급",
        amount: 500000
    },{
        category:"식비",
        detail:"서브웨이",
        amount:-30000
    },{
        category:"용돈",
        detail:"10월 용돈",
        amount:100000
    },{
        category:"식비",
        detail:"수육국밥",
        amount:-10000
    }];
```